docs/release-notes/2.38.0.md

```diff
@@ -0,0 +1,236 @@
+
+
+#### Sharded cluster MongoDB backup support
+
+This release introduces support for working with backups for sharded clusters. PMM handles the backup process end-to-end, but restoring such artifacts is currently possible only via the CLI, using Percona Backup for MongoDB.
+
+For information on restoring sharded backups, check the [PBM documentation](https://docs.percona.com/percona-backup-mongodb/usage/restore.html).
+
+
+### SQL comments support in QAN
+
+!!! caution alert alert-warning "Important/Caution"
+    This feature is still in [Technical Preview](https://docs.percona.com/percona-monitoring-and-management/details/glossary.html#technical-preview) and is subject to change. We recommend that early adopters use this feature for testing purposes only.
+
+You can now collect labels from the comments placed in SQL queries. You can tag the query by key value in the query comment. This helps in query management and optimization. This feature is currently supported for MySQL and PostgreSQL.
+
+
+For example:
+
+    ```sh
+    SELECT * /* cluster=’east’ */ FROM city;   
+    ```
+You can enable this feature as follows:
+
+CLI - Pass the parameter `comments-parsing`
+UI - It is enabled by default. To disable, uncheck the **Disable comments parsing** checkbox in the **Additional Options** section. This is supported for PostgreSQL and MySQL.
+
+![!image](../_images/PMM_QAN_Parsing.png)
+
+
+### New PMM Agent Down alert template
+
+We've added a new default alert template to [Percona Alerting](../get-started/alerting.md). 
+
+The new **PMM Agent down** alert template monitors your Node status to notify if the Agent is down. Before using this template:
+
+- ensure the PMM Agent is healthy and present on the Database Node.
+-  check that PMM Client is on version 2.38 version or later.
+
+![!image](../_images/Agent_down.png)
+
+## New Features
+
+- [PMM-2852](https://jira.percona.com/browse/PMM-2852) -   [QAN]: Starting with PMM 2.38.0, MySQL and PostgreSQL supports collection of labels from the comments placed in the SQL queries.
+- [PMM-11961](https://jira.percona.com/browse/PMM-11961) - [Inventory]: Starting with PMM 2.38.0, we have enhanced the **Inventory** page by improving the **Nodes** tab. With this change, you can get more context on your inventory objects, such as node, and more actionable information for exploring and resolving possible issues.
+- [PMM-11962](https://jira.percona.com/browse/PMM-11962) - [Inventory]: Starting with PMM 2.38.0, you can filter the nodes and services on the **Inventory** page using the parameters such as ID, names, status, etc., instead of scrolling through an exhaustive list of nodes or services.
+- [PMM-12087](https://jira.percona.com/browse/PMM-12087) - [Inventory]: Added tooltips to explain the meaning of the different services statuses in the Inventory view.   
```

Comment:
@rnovikovP, what do you think? Should we highlight inventory improvements?

Reply:
> @rnovikovP, what do you think? Should we highlight inventory improvements?

Aren't they highlighted already?

Reply:
oh, you are right. I missed that